src/frontend/src/icp/services/ic-add-custom-tokens.service.ts

```diff
@@ -53,22 +46,26 @@ export const loadAndAssertAddCustomToken = async ({
 		return { result: 'error' };
 	}
 
-	const { valid } = await assertLedgerId({
-		identity,
-		...canisterIds
-	});
+	const { valid } = isNullish(indexCanisterId)
```

Comment:
small suggestion, but completly up to you: I think it can be "cleaner" to have the invert it, meaning `nonNullish(....) ? ... : ...`

Reply:
Sure. Done.